Week3/week3_homework/exercise5.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const cartForParty = {
+  item_1: 0.99,
+  item_2: 3.5,
+  item_3: 10.3,
+  item_4: 1.75,
+  item_5: 33.46,
+};
+
+function calculateTotalPrice(items_obj) {
+  let total_price = 0;
+
+  for (const item in items_obj) {
+    total_price += items_obj[item];
+  }
+  return total_price;
```

Comment:
👍 for logic but apart from naming conventions which should be corrected, `reduce` which I mentioned before would be an ideal candidate here to sum all elements when used along with another newly introduced method in JavaScript called `Object.keys()`. Have a look. 🙂